be/src/olap/rowset/segment_v2/segment_iterator.h

```diff
@@ -159,6 +159,7 @@ class SegmentIterator : public RowwiseIterator {
     std::vector<ColumnId>
             _short_cir_pred_column_ids; // keep columnId of columns for short circuit predicate evaluation
     std::vector<bool> _is_first_read_column; // columns hold by segmentIter
+    std::vector<bool> _is_predicate_column;  // columns hold by segmentIter
```

Comment:
try not to use `vector<bool>`. Use `vector<char>` or `bitset` instead

Reply:
This vector will not be accessed frequently, so I think using `vector<bool>` is ok?